experimental/bits/simd_builtin.h

```diff
@@ -2284,7 +2284,8 @@ template <typename _Abi>
       const auto __absn = __vector_bitcast<_Ip>(_SuperImpl::_S_abs(__x));
       const auto __maxn
 	= __vector_bitcast<_Ip>(__vector_broadcast<_Np>(__finite_max_v<_Tp>));
-      return __absn <= __maxn;
+      return _MaskImpl::template _S_convert<_Tp>(
+	_MaskImpl::_S_to_bits(__as_wrapper<_Np>(__absn <= __maxn)));
```

Comment:
I don't understand how the compiler is able to reach this code when `_Abi` is `_VecBltnBtmsk`. One of the `if constexpr` cases in `_SimdImplX86::_S_isfinite` should be `true` and therefore the call to `_Base::_S_isfinite` not instantiated.
In most cases the generic implementation in `simd_builtin.h` should not be expected to return bit masks. Also, how does any `_VecBuiltin` code still compile after this change?

Reply:
as issue https://github.com/VcDevel/std-simd/issues/24 happened, I try to bypath the _SimdImplX86::_S_isfinite by call _Base::_S_isfinite directly that leads to the cases.
As far as I understand, the _Base::_S_isfinite works as the fallback case, it should be work as well, Is it?

Reply:
The `_SimdImplBuiltin` functions sometimes work for bitmasks and sometimes don't. Since, at this point, only AVX512 (i.e. x86) uses the bitmask ABI, I prefer to keep bitmask code out of the generic implementation. IIRC SVE also uses bitmasks. So once we have a SVE we should revisit how to abstract it best. Having the `_SimdImplBuiltin` functions support both may become a maintenance burden, decrease optimization, and increase compile times.
I any case, let's figure out why #24 happens and if this is really the proper solution.

Reply:
I see.
By the way, have you try to put SVE support on schedule?

Reply:
There's still so much else to do that I can't start on SVE. I don't have a schedule for it, no. :disappointed: